shared/design/src/stories/question.stories.tsx

```
@@ -9,30 +9,127 @@ const meta = {
   args: {
-    title:
-      "선택된 질문입니다선택된 질문입니다선택된 질문입니다선택된 질문입니다선택된 질문입니다선택된 질문입니다선택된 질문입니다선택된 질문입니다선택된 질문입니다선택된 질문입니다선택된 질문입니다선택된 질문입니다선택된 질문입니다선택된 질문입니다선택된 질문입니다",
     state: "error",
+    children: "placeholder" as React.ReactNode,
+  },
+  render: (args) => (
+    <Question state={args.state} onClick={args.onClick}>
+      <Question.Title>
+        질문은 입력되었지만 답안이 입력되지 않은 에러 상태입니다
+      </Question.Title>
+      <Question.Image />
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)